centreon/www/include/common/form/form.css

```
@@ -120,6 +120,14 @@
     filter: brightness(0.96);
 }
 
+/* The header is keyboard-operable, so its focus has to be visible. currentColor
+   rather than an accent: the header background is themed, and a fixed blue sits
+   at ~1:1 on the dark one. Inset, as the section clips its corners. */
+.cf-section-header:focus-visible {
+    outline: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)